SPECS/moby-cli/moby-cli.spec

```diff
@@ -80,6 +81,9 @@ install -p -m 644 contrib/completion/fish/docker.fish %{buildroot}%{_datadir}/fi
 %{_datadir}/fish/vendor_completions.d/docker.fish
 
 %changelog
+* Tue Jan 9 2024 corvus-callidus <108946721+corvus-callidus@users.noreply.github.com> - 20.10.27-1
```

Comment:
Version in changelog should be 20.10.27-2

Reply:
Thanks for catching that.